### DISCUSSION TITLE
Home Assitant binary sensor entity always show unavailable?

### DISCUSSION THREAD

**Discussion by Bubbgump209** [blakeblackshear/frigate#3171] [Ask A Question] [answered]
I want to use Frigate as a binary sensor (world's fanciest PIR of sorts, however the binary sensor entities always show as unavailable. I have confirmed that in Frigate it sees tracked objects (me). I have also confirmed that both Frigate and Home Assistant are happily connected to the same MQTT broker.  I deleted and reinstalled the integration to no effect. What am I missing?

**Comment by NickM-27** [comment]
Possible duplicate of https://github.com/blakeblackshear/frigate/issues/3166

**Reply by Bubbgump209** [reply]
Likely the case as I am on 2.2.1 and HACS doesn't show 2.2.2 as available. Thank you!

**Comment by Bubbgump209** [comment, marked as answer]
For anyone else, yes, manually installing v2.2.2 of the integration fixed everything.
